src/PodVector.cpp

```diff
@@ -90,13 +88,13 @@ PodVectorBase::GetNewCapacity(size_type minCapacity, size_type maxCapacity)
     return cap;
 }
 
-void* PodVectorBase::Reallocate(void* pb, size_t cb, bool zeroInitializeMemory)
+void* PodVectorBase::Allocate(size_t cb, bool zeroInitializeMemory)
 {
-    void* const pbNew = pb ? realloc(pb, cb) : malloc(cb);
+    void* const pbNew = malloc(cb);
```

Comment:
Note: In the realloc case with zeroInitializeMemory enabled, the memset would clobber the existing data. Since realloc doesn't really give a significant improvement over malloc+copy, just use malloc always instead.